<?xml version="1.0" encoding="utf-8"?>
<launch>

  <!-- Param -->

  <!-- Sensor -->
  <include file="$(find happymimi_bringup)/launch/sensor.launch" >
    <arg name="2dlidar" default="true" />
    <arg name="realsense" default="false" />
    <arg name="dynamixel" default="false" />
    <arg name="respeaker" default="false" />
  </include>

  <!-- AMCL -->
  <include file="$(find happymimi_navigation)/launch/include/gmapping.launch.xml"/>

  <!-- RViz -->
  <include file="$(find happymimi_rviz_launchers)/launch/view_navigation.launch"/>

  <!-- Joy teleop -->
  <include file="$(find happymimi_teleop)/launch/joycon_teleop.launch"/>

  <!-- ロケーション設定サーバーの起動 -->
  <node name="create_location" pkg="location_setup" type="create_location" />

</launch>
